patches/bullmq@3.5.1.patch

```diff
@@ -1,18 +1,5 @@
-diff --git a/dist/esm/classes/redis-connection.js b/dist/esm/classes/redis-connection.js
-index 7843e65..3807efe 100644
---- a/dist/esm/classes/redis-connection.js
-+++ b/dist/esm/classes/redis-connection.js
-@@ -105,7 +105,7 @@ export class RedisConnection extends EventEmitter {
-     }
-     loadCommands() {
-         return (this._client['bullmq:loadingCommands'] ||
--            (this._client['bullmq:loadingCommands'] = scriptLoader.load(this._client, path.join(__dirname, '../commands'))));
-+            (this._client['bullmq:loadingCommands'] = scriptLoader.load(this._client, path.join(__dirname, process.env.BULLMQ_COMMANDS_FROM_ROOT ? './' : '../commands'))));
```

Comment:
Did you test it on staging?

Reply:
I think v3 is no longer uses the `commands` in that form, but let me double check.

Reply:
If it works in integration tests, it will work on the cluster, but just wanted to make sure :)

Reply:
tested, all good ✅